Makefile

```diff
@@ -57,9 +57,17 @@ really-clean clean-all cleanall: clean
 
 procs := $(shell grep -c ^processor /proc/cpuinfo 2>/dev/null || echo 1)
 
-lint: vendor golangci-lint-config.yaml
+go-lint: vendor golangci-lint-config.yaml
 	@golangci-lint --version
 	golangci-lint run --config golangci-lint-config.yaml --new-from-rev origin/master --max-issues-per-linter 0 --max-same-issues 0
+.PHONY: go-lint
+
+tf-lint:
+	@terraform fmt -check -recursive -diff || \
```

Comment:
Is there a new command in the `tools` target, or documentation on where to get the terraform binary and if we have version dependencies?

Reply:
Don't have details about versions to support, But IaC team recommends terraform >= 0.13 and documented here https://github.com/HewlettPackard/hpegl-metal-terraform-resources#requirements